source/scenes/DayOf/scenes/Maps/index.jsx

```diff
@@ -1,7 +1,48 @@
 import React from 'react';
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
+import MapECEB from '../../../../assets/Maps/HackillinoisMAPS-ECEB.png';
```

Comment:
This can be shortened to  `'assets/Maps/HackillinoisMAPS-ECEB.png'`

Reply:
Shorten